kpi/002.adoc

```diff
@@ -1,12 +1,12 @@
-== B. Good quality title
+== KPI-{counter:kpi}: Good quality title
 
 === Status
 
 Approved
 
 === WCMP element(s)
 
-`/gmd:MD_Metadata/gmd:identificationInfo//gmd:citation/gmd:CI_Citation/gmd:title`
+* `/gmd:MD_Metadata/gmd:identificationInfo//gmd:citation/gmd:CI_Citation/gmd:title`
```

Comment:
line 13: The title field shall always be populated and ~and~ follow the principles of...